src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub fn give_up<T: 'static>() -> Box<T> {
 	};
 
 	for _ in 0..size {
-		v.push((rng.rand_u32() % 256) as u8);
+		v.push((rng.rand_u32() & Into::<u32>::into(u8::MAX)) as u8);
```

Comment:
```suggestion
		v.push((rng.rand_u32() & u32::from(u8::MAX)) as u8);
```
This is shorter and still without using `as`, if that's what you wanted to avoid.

Reply:
Thanks for the suggestion - I hope you believe me when I say I knew about `from` but mindlessly picked what the compiler suggested as an alternative to turbofishing.